fuel-jackson/build.gradle

```diff
@@ -0,0 +1,68 @@
+group 'com.github.kittiunf.fuel'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$parent.ext.kotlin_version"
+    }
+}
+
+
+apply plugin: 'java'
+apply plugin: 'kotlin'
+apply plugin: 'com.novoda.bintray-release'
+apply plugin: 'jacoco'
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+}
+
+compileKotlin {
+    kotlinOptions.jvmTarget = "1.8"
+}
+compileTestKotlin {
+    kotlinOptions.jvmTarget = "1.8"
+}
+
+
+sourceSets {
+    main.java.srcDirs += 'src/main/kotlin'
+    test.java.srcDirs += 'src/test/kotlin'
+}
+
+repositories {
+    jcenter()
+}
+
+dependencies {
+    compile "com.fasterxml.jackson.module:jackson-module-kotlin:$rootProject.jackson_version"
+    compile "org.jetbrains.kotlin:kotlin-stdlib:$parent.ext.kotlin_version"
+    compile project(':fuel')
+    testCompile "junit:junit:$parent.ext.junit_version"
+    testCompile group: 'org.hamcrest', name: 'hamcrest-all', version: '1.3'
```

Comment:
I think `hamcrest` is not needed...

Reply:
@kittinunf Well, hamcrest is already included with junit. This just adds a bunch of Matchers. But, sure I'll remove it.